Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -185,7 +185,7 @@ static float std_dev_points(float z_pt[NPP + 1], const bool _0p_cal, const bool
         S2 += sq(z_pt[rad]);
         N++;
       }
-      return round(SQRT(S2 / N) * 1000.0) / 1000.0 + 0.00001;
+      return LROUND(SQRT(S2 / N) * 1000.0) / 1000.0 + 0.00001;
```

Comment:
Is `LROUND` actually better here, since the return type is `float`?

Reply:
On AVR is does not matter. On ARM,  round would force a conversion from double->float, and the parameter of round() is double, so it also forces a conversion from float->double